pkg/config/config_template.yaml

```diff
@@ -334,22 +334,22 @@ api_key:
 #
 # If TLS enabled, you may enforce TLS verification here (defaults to true)
 #
-# syslog_tls_verify: yes
+# syslog_tls_verify: true
 #
 {{ end -}}
 {{- if .Autodiscovery }}
 # Autodiscovery
 #
 # Change the root directory to look at to get cgroup statistics. Useful when running inside a
 # container with host directories mounted on a different folder.
-# Default if environment variable "DOCKER_DD_AGENT" is set to "yes"
+# Default if environment variable "DOCKER_DD_AGENT" is set to "true"
```

Comment:
For now `yes` is the only valid value for `DOCKER_DD_AGENT`, see https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L565-L568

We could change the check to only check that the envvar is set, like [`IsKubernetes`](https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L565-L568), I'd be all for moving them all to `true` indeed

Reply:
I want to confirm there is zero chance someone changed it to false/no

Reply:
It is an undocumented internal envvar set in the Dockerfile (as it was in Agent5), I think it's fair to assume manually setting it is unsupported.
@hkaj do you think there's a risk here?

Reply:
I understand the env var doesn't make much sense in a false context, so for more precise language, could we change the comment to:
```
# Default if environment variable "DOCKER_DD_AGENT" is set
```

Reply:
It's entirely internal, it's safe to update. We could even update config.go to check for its presence, not its value.